### PR TITLE
Homework3: Oleg Galushko

### PR DESCRIPTION
**Pull request by ghost**
### PR checklist
Please make sure you've passed PR checklist
- [x] my PR passes CI checks (flake8 and pycodestyle).
- [x] my PR named according to guidelines.
- [x] my PR contains complete tasks descriptions for all tasks.
- [x] my PR made from the fork feature branch to appropriate branch.
- [x] my PR contains only code that solves appropriate homework tasks. Redundant 
files (configs, etc. ) are put to .gitignore.

### CODE REVIEW

File: src/homework3/task2.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+    :param city_name: имя города для поиска
+    :param country_dict: словарь содержащий соответствия {страна1: [города1], страна2: [города2]}
+    :returns: страна в которой находится город или None если такого нет
+    :rtype: str
+    """
+    for k, v in country_dict.items():
+        if city_name in v:
+            return k
+    return None
```
Comment (Shurke) [inline src/homework3/task2.py:39]: А для чего здесь `return None`?

File: src/homework3/task2.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+Программа получает на вход количество стран N.
+Далее идет N строк, каждая строка начинается с названия страны,
+затем идут названия городов этой страны. В следующей строке записано число M,
+далее идут M запросов — названия каких-то M городов, перечисленных выше.
+Выходные данные
+Для каждого из запроса выведите название страны, в котором находится данный город.
+"""
+
+
+def update_cities_dict(country_str, country_dict):
```
Comment (Shurke) [inline src/homework3/task2.py]: А чем эта задача хуже остальных, что нет тайп хинтов?)

File: src/homework3/task2.py (hunk 1 above)
Comment (ghost) [reply src/homework3/task2.py:39]: из PEP8: Be consistent in return statements. Either all return statements in a function should return an expression, or none of them should. If any return statement returns an expression, any return statements where no value is returned should explicitly state this as return None, and an explicit return statement should be present at the end of the function (if reachable).
Там же есть примеры, что отсутствие возврата как и использование return без None не является правильным код стайлом.

File: src/homework3/task2.py (hunk 2 above)
Comment (ghost) [reply src/homework3/task2.py]: будет исправлено

**Comment by Shurke** [comment]
Итоговая оценка - 9.

**Pull request closed without merging**